src/components/homePageNewsSpotlight/query.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { drupalClient } from '@/lib/drupal/drupalClient'
+import { NewsSpotlightData } from './formatted-type'
+import { deserialize, JsonApiResponse } from 'next-drupal'
+import { formatter as formatImage } from '@/components/mediaImage/query'
+import { getNestedIncludes } from '@/lib/utils/queries'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude([
```

Comment:
that's a nice way to build the includes, I just hardcoded them all into the string on the hero.